common/spdat.h

```diff
@@ -1298,7 +1298,7 @@ struct SPDat_Spell_Struct
 /* 016 */	uint32 buff_duration_formula; // -- DURATIONBASE
 /* 017 */	uint32 buff_duration; // -- DURATIONCAP
 /* 018 */	uint32 aoe_duration;	// sentinel, rain of something -- IMPACTDURATION
-/* 019 */	uint16 mana; // Mana Used -- MANACOST
```

Comment:
May want to change the comments after this to indicate the proper byte size due to these change, just have to add 2 bytes to everything after this.

Reply:
Comments are just a running column count of the CSV spells file.

Reply:
Ah, didn't even read them, some places we have bytes.

Reply:
Good to know, if I do encounter that! @Kinglykrab